fe1-web/jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   preset: 'react-native',
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  setupFilesAfterEnv: ["jest-extended"],
+  testPathIgnorePatterns: ["/node_modules/"],
+  testMatch: [ "**/__tests__/**/?(*.)+(spec|test).[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)" ],
```

Comment:
Perfect! 👍

Reply:
Actually, re-reading it, I feel I can just keep the second element, as the first is redundant with it.